### PR TITLE
tests for reservationHelper lwc

### PR DESCRIPTION
**Pull request by msrivastav13**
### What issues does this PR fix or reference?
tests for reservationHelper lwc
Also simplifies the if else logic with readable code
#<Insert GitHub Issue>

## The PR fulfills these requirements:

[ ] Tests for the proposed changes have been added/updated.
[ ] Code linting and formatting was performed.

### Functionality Before

<insert gif and/or summary>

### Functionality After

<insert gif and/or summary>

### CODE REVIEW

File: es-space-mgmt/main/default/lwc/reservationHelper/reservationHelper.js [hunk 1]
```diff
@@ -46,20 +46,20 @@ export default class ReservationHelper extends LightningElement {
...
     handleCustomerSelect(message) {
         if (message.tileType === 'customer') {
-            if (!this.flowStarted) {
-                this.flowStarted = true;
-                const choiceEvt = new CustomEvent('customerchoice', {
-                    detail: message.properties
-                });
-                this.dispatchEvent(choiceEvt);
-            } else if (this.flowStarted) {
+            if (this.flowStarted) {
```
Comment (msrivastav13) [inline es-space-mgmt/main/default/lwc/reservationHelper/reservationHelper.js:49]: simplifies if and else logic!

**Comment by msrivastav13** [comment]
@pozil The build is failing on this because of the org does not have my domain enabled? Do we know the org thats connected to prerelease?

**Comment by msrivastav13** [comment]
> @pozil The build is failing on this because of the org does not have my domain enabled? Do we know the org thats connected to prerelease?

**Comment by adityanaag3** [comment]
@msrivastav13  You also might want to check for negative use cases like I've done here based on @pozil 's feedback.. https://github.com/trailheadapps/easy-spaces-lwc/pull/137/files

Both these components kind of do the same thing.

**Comment by msrivastav13** [comment]
@adityanaag3 cool, will do some negative test cases as well!

**Comment by msrivastav13** [comment]
@adityanaag3 The negative test cases are done for this. @pozil This can be reviewed now!

**Review by pozil — APPROVED** [review]
Well done @msrivastav13, that was a tough one :)

**Pull request merged**